cmd/podman/containers/exec.go

```diff
@@ -87,7 +87,7 @@ func execFlags(cmd *cobra.Command) {
 	_ = cmd.RegisterFlagCompletionFunc(workdirFlagName, completion.AutocompleteDefault)
 
 	waitFlagName := "wait"
-	flags.Int32(waitFlagName, 0, "Total seconds to wait for container to start")
+	flags.Int32(waitFlagName, 0, "Total seconds to wait for container to start. Configurable using podmansh_timeout in containers.conf")
```

Comment:
Should the default be set to podmansh_timeout instead of 0?

Reply:
Right this sentence makes no sense, this --wait time-out is not configurable and IMO does not need to be. --wait is never used as default so there does not need to be a default value.

This only makes sense for podmansh which already sets this correctly so this changed comment here is incorrect and should be removed.

And also the flag is hidden so these comments are useless anyway as they will never appear in --help.

Reply:
fixed.